<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsComposerMapBase</class>
 <widget class="QWidget" name="QgsComposerMapBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>7</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Map options</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>10</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="mNameLabel" >
     <property name="text" >
      <string>&lt;b>Map&lt;/b></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line1" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1_3" >
     <property name="text" >
      <string>Set</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="mCalculateComboBox" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mWidthLineEdit" />
   </item>
   <item>
    <widget class="QLabel" name="textLabel2" >
     <property name="text" >
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mHeightLineEdit" />
   </item>
   <item>
    <widget class="Q3GroupBox" name="groupBox2" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Scale</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>10</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="textLabel1_6" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>1 :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mScaleLineEdit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mSetCurrentExtentButton" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Set map extent to current extent in QGIS map canvas</string>
     </property>
     <property name="text" >
      <string>Set Extent</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1_4" >
     <property name="text" >
      <string>Line width scale</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mWidthScaleLineEdit" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="toolTip" >
      <string>Width of one unit in millimeters</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1_2" >
     <property name="text" >
      <string>Symbol scale</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mSymbolScaleLineEdit" />
   </item>
   <item>
    <widget class="QLabel" name="textLabel2_2" >
     <property name="text" >
      <string>Font size scale</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mFontScaleLineEdit" />
   </item>
   <item>
    <widget class="QCheckBox" name="mFrameCheckBox" >
     <property name="text" >
      <string>Frame</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1_5" >
     <property name="text" >
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="mPreviewModeComboBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends></extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
